MobileCenter/MobileCenterTests/MSDeviceTrackerTests.m

```diff
@@ -120,14 +122,24 @@ - (void)testDeviceOSVersion {
   osSystemVersionMock.patchVersion = 6;
   OCMStub([processInfoMock operatingSystemVersion]).andReturn(osSystemVersionMock);
 #else
```

Comment:
Yeah but wanted to keep it because the next comment only applies to else case. Let me know if you like to remove it.